cors.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "origin": ["http://localhost:3000", "https://hoverboard-v2-dev.firebaseapp.com/"],
```

Comment:
Do you thinks we need to have this in config? It's would be used only once, when you have to run the commantd in gsutil mannually. Moreover this data is static, and does not processed by gulp. And I don't see any big reason that we have to put it there